examples/phase_change/ex4/LSLocateInterface.h

```diff
@@ -0,0 +1,102 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2017 - 2019 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+/////////////////////// INCLUDE GUARD ////////////////////////////////////
+
+#ifndef included_LSLocateInterface
+#define included_LSLocateInterface
+
+///////////////////////////// INCLUDES ///////////////////////////////////
+#include <ibamr/AdvDiffHierarchyIntegrator.h>
+
+namespace IBTK
+{
+class HierarchyMathOps;
+}
+
+/*
+ * Pre processing call back function to be hooked into IBAMR:LInitStrategy
```

Comment:
```suggestion
 * Pre processing call back function to be hooked into IBAMR::LSInitStrategy
```
This typo may also show up in other files.